core/chains/terra/terratxm/txm_test.go

```diff
@@ -31,7 +31,6 @@ import (
 )
 
 func TestTxm_Integration(t *testing.T) {
-	t.Skip() // Local only unless we want to add terrad to CI env
```

Comment:
Maybe upgrade to a `-short` skip instead?
```go
if testing.Short() {
	t.Skip("skipping due to use of FullTestDB")
}
```
Do we expect everyone to install terrad locally? Or should this test be CI only - with a build tag or something?

Reply:
Yeah should be CI only